index.js

```diff
@@ -96,7 +98,27 @@ module.exports = async params => {
 	}
 
 	const {packageJson} = readPkgUp.sync();
-	const {appId} = packageJson.build;
+	const buildConfig = packageJson.build;
+	let appId = '';
+
+	if (buildConfig) {
+		// appId from ./package.json
+		appId = buildConfig.appId;
+	} else {
+		try {
+			// fallback for vue-cli-plugin-electron-builder
```

Comment:
I suggest to remove the comment, as it's relevant not only to the vue plugin, but to all cases when electron config is done outside package.json.